examples/twittermap/web/public/javascripts/searchbar/controllers.js

```diff
@@ -2,9 +2,43 @@ angular.module('cloudberry.util', ['cloudberry.common'])
   .controller('SearchCtrl', function($scope, $window, cloudberry, cloudberryConfig) {
     $scope.search = function() {
       if ($scope.keyword && $scope.keyword.trim().length > 0) {
-        cloudberry.parameters.keywords = $scope.keyword.trim().split(/\s+/);
-        // skip the empty query for now.
-        cloudberry.query(cloudberry.parameters);
+
+          //stopword filtering feature
+          String.prototype.removeStopWords = function() {
+              var x;
+              var word;
+              var cleansed_string = this.valueOf();
+              var Hstop_words = external_stop_words();
```

Comment:
We should not call this every time when we do a search. Let's assign Hstop_words in $scope and use it.